web/packages/shared/components/AccessRequests/NewRequest/RequestCheckout/RequestCheckout.tsx

```diff
@@ -194,6 +194,7 @@ export function RequestCheckout<T extends PendingListItem>({
         <ArrowBack
           size="large"
           mr={3}
+          data-testid="close-checkout"
```

Comment:
I have a fix for the bulk add/remove in the web UI which is all mostly `e` code, but i had to add this testid and instead of making a new PR for that, I'm just going to hijack this one for the change if thats ok 🙏